src/output_parser/Oyente.py

```diff
@@ -32,7 +32,9 @@ def parse(self, str_output):
             elif "INFO:symExec:	  " in line:
                 (key, value) = self.extract_result_line(line)
                 current_contract[key] = value
-            elif current_contract is not None and "INFO:symExec:" + current_contract['file'] in line:
+            elif current_contract is not None and current_contract['file'] in line:
```

Comment:
replace: 

`current_contract is not None`

by 

`not current_contract`

Reply:
Why `not current_contract`? It should be just `current_contract`, right?

Reply:
correct, I stand corrected.

Reply:
You want me to change the others `current_contract is not None` to just `current_contract`?

Reply:
that would be awesome (but I suggest to use another PR).